test/domUpdates-test.js

```diff
@@ -1,5 +1,53 @@
+const chai = require('chai');
 import {expect} from 'chai';
-import domUpdates from '../src/domUpdates';
```

Comment:
I wonder why this was being weird!

Reply:
The test was trying to access another file as opposed to using the defined 'domUpdates'